modules/navier_stokes/doc/content/source/fvbcs/INSFVSwitchableOutletPressureBC.md

```diff
@@ -0,0 +1,32 @@
+# INSFVSwitchableOutletPressureBC
+
+!syntax description /FVBCs/INSFVSwitchableOutletPressureBC
+
+The file is similar to [INSFVOutletPressureBC.md] but it allows the boundary conditions to be switched on/off.
+
+This object simply wraps [`FVFunctionDirichletBC`](FVFunctionDirichletBC.md), so
```

Comment:
Need to update this documentation

Reply:
Actually I see this is a copy of the now-base-class doc, so you can just delete this whole paragraph.

Reply:
Thanks. Deleted